cmds/coredhcp-generator/main.go

```diff
@@ -107,13 +108,16 @@ func main() {
 		log.Fatalf("No plugin specified!")
 	}
 	outfile := *flagOutfile
+	var tmpdir string
 	if outfile == "" {
-		tmpdir, err := ioutil.TempDir("", "coredhcp")
+		var err error
```

Comment:
since you removed the go.mod copying I think this change block isn't necessary anymore

Reply:
good catch